bears/general/CPDBear.py

```diff
@@ -5,6 +5,7 @@
 from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
 from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.aspects.Redundancy import Redundancy
 
 
 class CPDBear(GlobalBear):
```

Comment:
@Asnelchristian Hi :) I'm wondering if we could do something like:

```python
class CPDBear(GlobalBear, aspects=[...]):
    ...
```

@sils Would look fancy, right? :D

Let's try to implement it with some metaclass magic... As aspect team task!